yt_dlp/extractor/servus.py

```diff
@@ -63,86 +62,51 @@ class ServusIE(InfoExtractor):
     def _real_extract(self, url):
         video_id = self._match_id(url).upper()
 
-        token = self._download_json(
-            'https://auth.redbullmediahouse.com/token', video_id,
-            'Downloading token', data=urlencode_postdata({
-                'grant_type': 'client_credentials',
-            }), headers={
-                'Authorization': 'Basic SVgtMjJYNEhBNFdEM1cxMTpEdDRVSkFLd2ZOMG5IMjB1NGFBWTBmUFpDNlpoQ1EzNA==',
-            })
-        access_token = token['access_token']
-        token_type = token.get('token_type', 'Bearer')
+        # TODO: timezone should be IANA tz database name (`Intl.DateTimeFormat().resolvedOptions().timeZone` with JavaScript)
+        # but api server only seems to check the format with regex `^([A-Z][A-Za-z]+)(\/.+)?$` for now
+        timezone = 'Etc/GMT%+d' % int(time.timezone / 3600)
+        attrs = self._download_json(
+            'https://api-player.redbull.com/stv/servus-tv?videoId=%s&timeZone=%s' % (video_id, timezone),
+            video_id, 'Downloading video JSON')
 
-        video = self._download_json(
-            'https://sparkle-api.liiift.io/api/v1/stv/channels/international/assets/%s' % video_id,
-            video_id, 'Downloading video JSON', headers={
-                'Authorization': '%s %s' % (token_type, access_token),
-            })
+        if 'GEO_BLOCKED' in attrs.get('playabilityErrors', ''):
+            countries = ', '.join(attrs.get('blockedCountries', ['Unknown']))
+            raise ExtractorError(
+                'Video is geo restricted (restricted countries: %s). '
+                'Try bypassing with --geo-bypass-country option, VPN or --proxy option' % countries, expected=True)
```

Comment:
```suggestion
        if 'GEO_BLOCKED' in attrs.get('playabilityErrors', []):
            self.raise_geo_restricted(countries=attrs.get('blockedCountries'))
```
Atleast, that's what I think restricted countries means? You might want to also check for the other errors